transporte.py

```diff
@@ -50,10 +50,9 @@
     nova_geracao.append(melhor_individuo) # Elitismo - nova geração começa com o melhor individuo
 
     while len(nova_geracao) < (TAMANHO_POPULACAO / 5): # 20% da nova geração é filha dos 10 melhores individuos da geração anterior
-        pai1_fitness, pai2_fitness = random.choices(populacao_fitness[:10], k=2)
-
-        pai1 = pai1_fitness[0]
-        pai2 = pai2_fitness[0]
+        
+        #pai1, pai2 = metodo_selecao_aleatorio(populacao_fitness)  # Seleção aleatória de pais
```

Comment:
Aqui podemos usar os 2 métodos, aleatório ou torneio, definido por um int random. Se for impar usa o aleatório, se for par usa o torneio